src/server.ts

```diff
@@ -0,0 +1,71 @@
+import * as grpc from 'grpc';
+import * as protoLoader from '@grpc/proto-loader';
+import * as apm from 'elastic-apm-node';
+import { GrpcHealthCheck, HealthCheckResponse, HealthService } from 'grpc-ts-health-check';
+import { config } from './config';
+import { log, Severity } from './utils/logger';
+import { wrapper } from './utils/wrapper';
+import { DropboxMethods } from './dropbox/dropbox.methods';
+
+apm.start({
+    serviceName: config.serviceName,
+    secretToken: config.apm.secretToken,
+    verifyServerCert: config.apm.verifyServerCert,
+    serverUrl: config.apm.apmURL,
+});
+
+export const serviceNames: string[] = ['', 'dropbox.dropboxService'];
+
+export class Server {
+
+    private dropbox_proto: any;
+    public server: grpc.Server;
+    public grpcHealthCheck: GrpcHealthCheck;
+
+    public constructor(address: string) {
+        this.server = new grpc.Server();
+        this.grpcHealthCheck = new GrpcHealthCheck({
+            '': HealthCheckResponse.ServingStatus.UNKNOWN,
+            serviceName: HealthCheckResponse.ServingStatus.UNKNOWN
+        });
+
+        this.initialProto();
+        this.addServices();
+        this.server.bind(address, grpc.ServerCredentials.createInsecure());
+        log(Severity.INFO, `server listening on address: ${address}`, 'server bind');
+    }
+
+    private initialProto() {
```

Comment:
Maybe `initiateProto`  is a more suitable name.

Reply:
done.